src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskList {
+    ArrayList<Task> store;
```

Comment:
Any reason using `store` instead of the plural form of task e.g. `tasks` ? I feel `tasks` would be more intuitive, just a suggestion.